docs/community.md

```diff
@@ -37,6 +37,8 @@ application we use most often to interact with the Matrix protocol is the
   Room for validators learning about setting up a node.
 - [Polkadot Direction](https://app.element.io/#/room/!OwgojQyBzTlUQGGLhq:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation) -
   Governance, and a place to discuss the future of Polkadot.
+- [Polkadot Digest](https://matrix.to/#/!vMpYyTkvjXcevxSdsQ:web3.foundation?via=web3.foundation&via=matrix.org&via=matrix.parity.io) -
```

Comment:
```suggestion
- [Polkadot Digest](https://matrix.to/#/!vMpYyTkvjXcevxSdsQ:web3.foundation) -
```

I don't know what the `&via` tags do - the link works without them.

Reply:
I just had Matrix give me the link, didn't investigate too closely.  but better to get rid of it if not needed